mapstore/localConfig.json

```diff
@@ -0,0 +1,606 @@
+{
+    "proxyUrl": {
+        "url": "proxy/?url=",
+        "useCORS": [
+          "https://georchestra.geo-solutions.it"
+        ],
+        "autoDetectCORS": true
+    },
+    "geoStoreUrl": "rest/geostore/",
+    "printUrl": "pdf/info.json",
+    "bingApiKey": "AhuXBu7ipR1gNbBfXhtUAyCZ6rkC5PkWpxs2MnMRZ1ZupxQfivjLCch22ozKSCAn",
+    "mapquestApiKey": "__API_KEY_MAPQUEST__",
+    "initialMapFilter": "",
+    "ignoreMobileCss": false,
+    "useAuthenticationRules": true,
+    "loadAfterTheme": true,
+    "header": {
+      "height": 90,
+      "url": "/header/"
+    },
+    "defaultMapOptions": {
+      "cesium": {
+          "flyTo": true,
+          "navigationTools": true,
+          "terrainProvider": {
+              "type": "ellipsoid"
+          }
+      }
+    },
+    "authenticationRules": [{
+        "urlPattern": ".*geostore.*",
+        "method": "bearer"
+    }, {
+        "urlPattern": ".*rest/config.*",
+        "method": "bearer"
+    }],
+    "monitorState": [
+      {"name": "router", "path": "router.location.pathname"},
+      {"name": "browser", "path": "browser"},
+      {"name": "geostorymode", "path": "geostory.mode"},
+      {"name": "featuregridmode", "path": "featuregrid.mode"}],
+    "projectionDefs": [{
+      "code": "EPSG:2154",
```

Comment:
These are projections commonly used in France.
I know geOrchestra is mostly used in France, but shouldn't we have something more international here by default ?

Reply:
@fvanderbiest this is what has been asked to us, we can add as many projection as you like

Reply:
ping @landryb what do you think ? mapfishapp was configured with international defaults.

Reply:
sure, yet most users so far are in france, so they'll have to adapt their configs (and/or pester their contractors to do so ;) @catmorales what do you think ?